src/class-translation-queue.php

```
@@ -64,9 +64,44 @@ private function __construct() {
+     * Ensure recurring Action Scheduler actions exist.
+     *
+     * Hooked to 'init' at priority 10 (after AS's DB store at priority 1)
+     * so the store is ready when we schedule. Runs on every page load to
+     * self-heal after activation, database restores, or accidental deletion.
+     *
+     * @since 1.1.1
+     * @return void
+     */
+    public function ensure_scheduled_actions(): void {
+        if ( false === as_next_scheduled_action( 'gratis_ai_ts_process_queue', [], 'gratis_ai_ts' ) ) {
+            as_schedule_recurring_action( time(), 5 * MINUTE_IN_SECONDS, 'gratis_ai_ts_process_queue', [], 'gratis_ai_ts' );
```

PHPStan (2.1.46)

[error] Constant MINUTE_IN_SECONDS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)